src/metatrader/factories/context_factory.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Why is the context factory in its own folder? We might have other APIs and they will also require initializations so shouldn't this sit with all of the other factories?